src/main/java/core/basesyntax/state/Figure.java

```diff
@@ -0,0 +1,27 @@
+package core.basesyntax.state;
+
+import core.basesyntax.behavior.FigureArea;
+import core.basesyntax.behavior.FigureInformation;
+
+public abstract class Figure implements FigureInformation, FigureArea {
+    private String name;
+    private String color;
+
+    public Figure(String name) {
+        this.name = name;
+        color = ColorSupplier.getRandomColor();
```

Comment:
Take color from the constructor

Reply:
Why write it in each type of Figure if I could write it like that in Figure constructor once?

Reply:
changed